src/Sarif/Autogenerated/Notification.cs

```diff
@@ -41,13 +41,13 @@ public SarifNodeKind SarifNodeKind
         public string Id { get; set; }
 
         /// <summary>
-        /// The stable, unique identifier of the rule (if any) to which this notification is relevant. This member can be used to retrieve rule metadata from the rules dictionary, if it exists.
+        /// The stable, unique identifier of the rule, if any, to which this notification is relevant.
```

Comment:
>if any [](start = 55, length = 6)

This was lost in the schema merge.

Reply:
Sure. Would materialize on next BuildAndTest gesture. It's important to perform this step before finalizing a push to get relevant changes in autogen code, sorry for omitting this step.

---
In reply to: [250303030](https://github.com/Microsoft/sarif-sdk/pull/1232#discussion_r250303030) [](ancestors = 250303030)